app/valid.py

```diff
@@ -0,0 +1,27 @@
+from flask import abort, make_response
+import requests
+import os
+
```

Comment:
I love this `valid` package full of your helper methods and validation methods! Fantastic idea and well done.